examples/slurm/prologMoneo.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo "[worker]" > /tmp/jobid_$SLURM_JOB_ID.txt
+echo $SLURM_JOB_NODELIST >> /tmp/jobid_$SLURM_JOB_ID.txt
```

Comment:
These can be error prone. Please see if we can avoid the worker header.

Reply:
I will look into this.

Reply:
I have addressed this